doc/src/manual/types.md

```diff
@@ -711,6 +714,17 @@ julia> Pointy{Real} <: Pointy{Float64}
 false
 ```
 
+The notation `Pointy{<:Real}` can be used to express the Julia analogue of a
+*covariant* type, while `Pointy{>:Int}` the analogue of a *contravariant* type,
```

Comment:
Worth linking to https://en.wikipedia.org/wiki/Covariance_and_contravariance_(computer_science), do you think?